src/apps/mesoscale-explorer/ui/states.tsx

```diff
@@ -299,7 +300,45 @@ export const LoadModel = StateAction.build({
     }
 }));
 
-//
+
+export const LoadCellPackColorsAction = StateAction.build({
```

Comment:
Doing this as a state action only makes sense when this would be added to the UI via the plugin spec, which isn't the case here. This could be a simple function that accepts the `File` and `PluginContext`.

Reply:
Is there examples I could find that show how to do it more simply ?

Reply:
just make it a normal `function` which you call.